### ISSUE TITLE
why do not use model.eval() for pointnet?

### ISSUE THREAD

**Issue by hwjiang1510** [samarth-robo/contactdb_prediction#5]
as title

**Comment by samarth-robo** [comment]
We found that it causes instabilities in the BatchNorm layers. This has been [documented before](https://discuss.pytorch.org/t/model-eval-gives-incorrect-loss-for-model-with-batchnorm-layers/7561/2). Another work around is to forward pass some samples and then setting `model.eval()`.
